apps/catalogue/src/views/ResourceDetailsView.vue

```diff
@@ -1,130 +1,30 @@
 <template>
-  <div class="container bg-white" id="_top" v-if="row">
```

Comment:
I originally chose 'row' with the idea that we could generalize this to all types of rows.

Reply:
row confused me in this context as the context is a single row , and we are redering row's of cards and list items , suggest to use row and column when referring to tables, and use section and field ( or something else , but not row and column) when the context is a cards ( data object)